product_docs/docs/pgd/5/reference/nodes-management-interfaces.mdx

```diff
@@ -595,3 +595,11 @@ bdr.alter_node_option(node_name text,
 -   `node_name` &mdash; Name of the node to be changed.
 -   `config_key` &mdash; Key of the option in the node to be changed.
 -   `config_value` &mdash; New value to be set for the given key.
+
+The node options that can be changed using this function are:
+- `route_priority` &mdash; Relative routing priority of the node against other nodes in the same node group. Default is '100'.
+- `route_fence` &mdash; Whether the node is fenced from routing; when true, the node can't receive connections from PGD Proxy. Default is 'f' (false).
+- `route_writes` &mdash; Whether writes can be routed to this node, that is, whether the node can become write leader. Default is 't' (true).
+- `route_reads` &mdash; Boolean: Whether read-only connections can be routed to this node. Currently reserved for future use. Default is 't' (true).
```

Comment:
Default is `t` for data and subscriber-only nodes else `f` for other node types like witness, standbys